dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/microprofile/metrics50/internal/SmallryeMetricsCDIMetadata.java

```diff
@@ -288,4 +304,23 @@ private void loadSmallRyeClasses(ClassLoader classLoader) throws ClassNotFoundEx
         Util.SR_APPLICATION_NAME_RESOLVER_INTERFACE = ApplicationNameResolverFuncInterface;
 
     }
+
+    @FFDCIgnore(ClassNotFoundException.class)
+    private void checkPrometheusRegistryAvailable(ClassLoader classLoader) {
+        // Check if Prometheus Meter Registry is in class path
+        try {
+            Class.forName(FQ_PROMETHEUSCONFIG_PATH, false, classLoader);
+            if (!Boolean.parseBoolean(ConfigProvider.getConfig()
```

Comment:
it seems like we'd only want to try to load the prometheus registry if the config indicates it's needed.  So we should do the config check before trying to load it.